`master` and `v4.3.0` branches on macos fails to build

As CI https://github.com/libressl/portable/actions/runs/24613001835 fails, it fails on my local env with the following error:

```
../../../tls/tls_keypair.c:151:26: error: use of undeclared identifier 'INT_MAX'
        if (keypair->cert_len > INT_MAX) {
                                ^
1 error generated.
```

It seems that the following patch (adding `<limits.h>` into `tls_keypair.c`) fixes the issue.

```
$ git diff
diff --git a/src/lib/libtls/tls_keypair.c b/src/lib/libtls/tls_keypair.c
index 8b1beda11..5557775bb 100644
--- a/src/lib/libtls/tls_keypair.c
+++ b/src/lib/libtls/tls_keypair.c
@@ -19,6 +19,8 @@
 #include <openssl/err.h>
 #include <openssl/pem.h>

+#include <limits.h>
+
 #include <tls.h>

 #include "tls_internal.h"
```

--- a/patches/tls_keypair.c.patch
+++ b/patches/tls_keypair.c.patch
@@ -1,0 +1,11 @@
+--- tls/tls_keypair.c.orig	Sun Apr 19 02:12:25 2026
++++ tls/tls_keypair.c	Sun Apr 19 02:12:33 2026
+@@ -15,6 +15,8 @@
+  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+  */
+ 
++#include <limits.h>
++
+ #include <openssl/bio.h>
+ #include <openssl/err.h>
+ #include <openssl/pem.h>

